torch/_subclasses/fake_tensor.py

```diff
@@ -466,9 +501,10 @@ def merge_devices(t):
 
 
 class FakeTensorMode(TorchDispatchMode):
-    def __init__(self, allow_fallback_kernels=True):
+    def __init__(self, *, allow_fallback_kernels=True, allow_meta=False):
```

Comment:
nice... is it generally good practice to have keyword-only args ? just allows more flexibility with respect to changes in the future basically ?

Reply:
I prefer kwargs to avoid boolean blindness; if I write `FakeTensorMode(True)` what does it mean? Can't tell without looking at the call site.